ethcrypto/ethcrypto.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Celer Network
+
+package ethcrypto
+
+import (
+	"crypto/ecdsa"
+	"encoding/hex"
+	"math/big"
+
+	"github.com/celer-network/goutils/log"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+type Signer struct {
+	key     *ecdsa.PrivateKey
+	chainId *big.Int
+}
+
+func NewSigner(privateKey string, chainId *big.Int) (*Signer, error) {
```

Comment:
nit: add NewSignerFromKeystore(ks, pwd string, chainId *big.Int)
ks is json string of keystore file content

Reply:
done